---
float32 bending_compliance
